tests/unit/simulation/test_flight_3dof.py

```diff
@@ -137,3 +137,176 @@ def test_invalid_simulation_mode(example_plain_env, calisto):
             rail_length=1,
             simulation_mode="2 DOF",
         )
+
+
+def test_weathercock_coeff_stored(example_plain_env, point_mass_rocket):
+    """Tests that the weathercock_coeff parameter is correctly stored.
+
+    Parameters
+    ----------
+    example_plain_env : rocketpy.Environment
+        A basic environment fixture for flight simulation.
+    point_mass_rocket : rocketpy.PointMassRocket
+        A point mass rocket fixture for 3-DOF simulation.
+    """
+    flight = Flight(
+        rocket=point_mass_rocket,
+        environment=example_plain_env,
+        rail_length=1,
+        simulation_mode="3 DOF",
+        weathercock_coeff=2.5,
+    )
+    assert flight.weathercock_coeff == 2.5
+
+
+def test_weathercock_coeff_default(example_plain_env, point_mass_rocket):
+    """Tests that the default weathercock_coeff is 1.0.
+
+    Parameters
+    ----------
+    example_plain_env : rocketpy.Environment
+        A basic environment fixture for flight simulation.
+    point_mass_rocket : rocketpy.PointMassRocket
+        A point mass rocket fixture for 3-DOF simulation.
+    """
+    flight = Flight(
+        rocket=point_mass_rocket,
+        environment=example_plain_env,
+        rail_length=1,
+        simulation_mode="3 DOF",
+    )
+    assert flight.weathercock_coeff == 1.0
+
+
+def test_weathercock_zero_gives_fixed_attitude(example_plain_env, point_mass_rocket):
```

Comment:
for the `test_weathercock_zero_gives_fixed_attitude`, can you re-use another fixture? 

fixtures make tests faster as pylint re-use their valuies instead of creating the object multiple times